src/reduxTokenAuthConfig.js

```diff
@@ -0,0 +1,25 @@
+import { generateAuthActions } from 'redux-token-auth'
+
+const config = {
+  authUrl: "https://glocal-news.herokuapp.com/api/v1/auth",
+  userAttributes: {
+    uid: 'uid'
+  },
+  userRegistrationAttributes: {
+    password_confirmation: 'password_confirmation'
+  },
+}
+
+const {
+  registerUser,
+  signInUser,
+  signOutUser,
+  verifyCredentials,
+} = generateAuthActions(config)
```

Comment:
just out of interest: these functions are all pre-defined in Redux Token Auth or?

Reply:
yes they are